### PR TITLE
Fix update

### PR DESCRIPTION
**Pull request by pape-io**

### CODE REVIEW

File: spec/item/save_spec.rb [hunk 1]
```diff
@@ -44,6 +44,10 @@ class SomeService < LHS::Service
       item.save
       expect(item.name).to eq 'Steve'
     end
+
+    it 'does not take backend values if perstiting fales' do
+
```
Comment (acoiro) [inline spec/item/save_spec.rb]: nice

File: spec/item/update_spec.rb [hunk 2]
```diff
@@ -38,10 +38,18 @@ class SomeService < LHS::Service
...
+        .to_return(status: 200, body: item._raw.merge(likes: 'Banana').to_json)
       item.update(name: 'Steve')
       expect(item.likes).to eq 'Banana'
     end
+
+    it 'updates local version of an object even if BE request fails' do
+      stub_request(:post, item.href)
+        .to_return(status: 400, body: item._raw.merge(likes: 'Banana').to_json)
+      item.update(name: 'Andrea')
+      expect(item.name).to eq 'Andrea'
```
Comment (acoiro) [inline spec/item/update_spec.rb]: lol

**Comment by acoiro** [comment]
:+1: merge!

**Comment by acoiro** [comment]
jenkins is faster!!!

File: spec/item/save_spec.rb (hunk 1 above)
Comment (pape-io) [inline spec/item/save_spec.rb]: removed

**Pull request merged**